src/translations/qjackctl_fr.ts

```diff
@@ -11,7 +11,7 @@
     <message>
         <location filename="../qjackctlInterfaceComboBox.cpp" line="157"/>
         <source>Please wait, PortAudio is probing audio hardware.</source>
-        <translation>Merci de patienter, PortAudio détecte le matériel audio.</translation>
+        <translation>PortAudio détecte le matériel audio, merci de patienter.</translation>
```

Comment:
Why this? It doesn't add anything and it reverse the original flow.

Reply:
Because that's the correct wording order for french IMHO.

Reply:
Both ways are correct as far as I know.

Reply:
I'd like this change to be reverted if you don't mind.

Reply:
On the way.